### PR TITLE
New checkboxes

### PR DESCRIPTION
**Pull request by noahmanger**
## Summary
New checkbox styles based on https://github.com/18F/fec-style/issues/314

Resolves https://github.com/18F/fec-style/issues/320

## Screenshots
image

_cc @username_

### CODE REVIEW

**Comment by noahmanger** [comment]
Ok! Ready for review @jenniferthibault .

**Comment by jenniferthibault** [comment]
😞 either I am having issues with my local env or something's not right. I see CodeCov is showing errors, so maybe it's that?

` && npm run less && cd .. rm -rf styleguide && kss-node --config kss-config.json && npm run copy-css'.
npm ERR! This is most likely a problem with the fec-style package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     cd fec-template/ && npm run less && cd .. rm -rf styleguide && kss-node --config kss-config.json && npm run copy-css`

I'm not seeing background colors:
screen shot 2016-04-29 at 7 06 31 pm
screen shot 2016-04-29 at 7 06 36 pm

**Comment by noahmanger** [comment]
Aaaand that would be because I didn't commit the changes 🙈 . Try now.

**Comment by jenniferthibault** [comment]
That makes more sense now 😄 

Ok! I made some small color adjustments and gave the box areas a little more space on the right edge. Looking good on the stylesheet and filter panel:

screen shot 2016-05-02 at 10 12 13 am

screen shot 2016-05-02 at 10 12 18 am

screen shot 2016-05-02 at 10 14 32 am


### But I also found an oops!

Looks like the style is somehow affecting our custom drop-down styles:

screen shot 2016-05-02 at 10 01 12 am

screen shot 2016-05-02 at 10 02 22 am

**Comment by noahmanger** [comment]
Changes look good. But yikes! Good catch. Fixed:

gif

Note that the focus state changed as part of this change, I think for the better. Previously, items that were in focus just were highlighted. Now, I removed the highlight, but kept the blue focus ring, which makes it clearer it's focused and not selected.

**Comment by noahmanger** [comment]
Hmmm apparently wifi at FEC isn't letting me push to GitHub, so that change isn't in yet. Will sync after I'm on a different network.

**Comment by jenniferthibault** [comment]
Awesome, looks good on my end! 

@adborden would you like to take a peek for technical review?

File: package.json [hunk 1]
```diff
@@ -7,12 +7,11 @@
     "url": "git://github.com/18F/fec-style.git"
   },
   "scripts": {
-    "build": "npm run build-sass && npm run build-js && npm run build-styleguide && npm run copy-css && npm run copy-images",
+    "build": "npm run build-sass && npm run build-js && npm run build-styleguide && npm run copy-images",
     "build-js": "browserify -t hbsfy --debug fec-template/public/init.js > fec-template/public/main.js",
     "build-sass": "node-sass scss/ -o css/",
     "build-styleguide": "cd fec-template/ && npm run less && cd .. rm -rf styleguide && kss-node --config kss-config.json && npm run copy-css",
```
Comment (adborden) [inline package.json]: Still a reference to `copy-css` here, so `npm run build-styleguide` fails. Should that just be removed?

**Comment by noahmanger** [comment]
Thanks! Fixed @adborden .

**Pull request merged**